hbase-common/src/main/java/org/apache/hadoop/hbase/zookeeper/ZKConfig.java

```diff
@@ -88,7 +108,7 @@ private static Properties extractZKPropsFromHBaseConfig(final Configuration conf
    * @return Properties holding mappings representing ZooKeeper config file.
    */
   private static Properties makeZKPropsFromHbaseConfig(Configuration conf) {
```

Comment:
Since this method is private static, we can give it a more reasonable name like makeZKServerPropsFromHBaseConfig?

Reply:
Done.